codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/protocols/core/AwsHttpBindingProtocolGenerator.kt

```diff
@@ -2,18 +2,18 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
-package software.amazon.smithy.kotlin.codegen.aws.protocols.core
+package software.amazon.smithy.kotlin.codegen.protocols.core
```

Comment:
Why did this and other packages change?

Reply:
Can you clarify why we need to move the AWS protocols from `smithy-aws-kotlin-codegen` to `smithy-kotlin-codegen`? IIRC this was an intentional split to separate AWS protocols from generic Smithy ones (CBOR). Customers could still add a dependency on the `smithy-aws-kotlin-codegen` module if they want to use an AWS protocol

Reply:
I think we should find a way to work around the circular dependency without relocating our AWS protocols. It was a design decision to keep them separate, if we no longer think that's the right choice, then we should have a discussion offline about moving them